### DISCUSSION TITLE
Welcome to UMatter Discussions!

### DISCUSSION THREAD

**Discussion by MonalikaPatnaik** [MonalikaPatnaik/UMatter#343] [Announcements]
## 👋 Welcome!
Hey all, starting **UMatter Github Discussions**😊 to facilitate conversations for the next 3 months. We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share your ideas.
  * Help and engage with other contributors.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.
  * Make sure that you follow [Code of Conduct](https://github.com/MonalikaPatnaik/UMatter/blob/main/CODE_OF_CONDUCT.md) and have read [Contributor flow](https://github.com/MonalikaPatnaik/UMatter/blob/main/CONTRIBUTING.md).

### A brief about myself!

I am an Outreachy intern at [OpenfoodFacts](https://github.com/openfoodfacts)![image](https://github.com/MonalikaPatnaik/UMatter/assets/99353300/f66f2134-8d85-4ff8-bc5c-db5e99238def) for May-August Internship Cohort and have to devote atleast 5 to 6 hours to complete my internship tasks. So, if the issues are not assigned to you or your PRs are not merged, just don't worry...be patient..we get back to you soon and reply each and everyone..

> Remember Open-Source is all about community bonding and collaboration.

If you think you can help other contributors with their questions you are free to do here☺️
We appreciate your Valuable Contributions and always looking forward to it🤩

**Comment by Kalyan-velu** [comment]
Hey please remove `/UMatter` from the live project link in description.

**Reply by MonalikaPatnaik** [reply]
Done!
Thankyou:-)
